scripts/testdata/satellite-config.yaml.lock

```diff
@@ -157,6 +157,9 @@ kademlia.operator.wallet: ""
 # uri which is used when retrieving new access token
 # mail.token-uri: ""
 
```

Comment:
I think we also need to add that new config for test data @fuskovic

Reply:
done